testsuite/features/step_definitions/common_steps.rb

```diff
@@ -1448,3 +1448,22 @@ def server_secret
   command = "sed -i '$d' /etc/hosts && sed -i '$d' /etc/hosts"
   $server.run(command)
 end
+
+# select an item from any dropdown
+When(/^I select "(.*?)" from "([^"]*)" dropdown/) do |selection, label|
+  # let the the select2js box filter open the hidden options
+  xpath_query = "//select[@name='#{label}']"
+  raise "xpath: #{xpath_query} not found" unless find(:xpath, xpath_query).click
+  # select the desired option
+  raise "#{label} #{selection} not found" unless find(:xpath, "//select[@name='#{label}']/option[contains(text(), '#{selection}')]").click
+end
```

Comment:
I suggest to move this step to navigation_steps or either close to other similar steps in this file (around line 520ish)